core/tests/webdriverio_utils/ClassroomPage.js

```diff
@@ -40,6 +42,10 @@ var ClassroomPage = function() {
       expect(topicSummaryTiles.length).toEqual(0);
     }
   };
+
+  this.takeDiagnosticTest = async function() {
+    await action.click('Take diagnostic test', takeDiagnosticTest);
```

Comment:
Like the selector name, the element description should be phrased as a noun

Reply:
Updated